### ISSUE TITLE
Unreal Gold 226 Crash on Startup

### ISSUE THREAD

**Issue by shoober420** [mmdanggg2/D3D9DrvRTX#8]
When trying to launch Unreal Gold 226 with the RTX render, I crash with errors. Heres a screenshot of the errors with logs and a crash dump in a zip file.

![20240908_145520.jpg](https://github.com/user-attachments/assets/562186ff-f523-4c55-ab18-6eaa34099cef)

![20240908_145457.jpg](https://github.com/user-attachments/assets/66d5fdbb-fb6b-423c-a251-faf6c71504e5)

![d3d9.log.zip](https://github.com/user-attachments/files/16923849/d3d9.log.zip)

Windows 11 27695
RTX Remix 0.5.4
i9 13900kf @5.8ghz all cores
NVIDIA RTX 4090 | 560.94

**Comment by mmdanggg2** [comment]
That's an error caused by RTX remix, try reinstalling it or you could try a different version.

**Comment by shoober420** [comment]
I made an issue on the NVIDIA RTX Remix gitub.

https://github.com/NVIDIAGameWorks/rtx-remix/issues/608

**Comment by shoober420** [comment]
I thought I would post an update. Im still getting this error with your new 1.4.0 RTX render and 565.90 NVIDIA drivers. This is on the latest RYX Remix 0.5.4 version.

**Comment by shoober420** [comment]
I just found out what it is. Something was fixed in the meantime for sure, either with the new 565.90 drivers or your new 1.4.0 RTX render release.

As you probably know, Unreal defaults to 640x480x16bit color when first starting the game. When I was testing prior, changing to 32bit color produced an orange rectangle at the top right of the screen and the rest was a black screen with the game running and sound playing. Using 16bit color crashed the game with the error posted earlier.

Now, with 1.4.0/565.90 and 32bit color, the game loads and renders normally. 16bit color still crashes the game, so we can chalk this up as an issue with 16bit color and RTX Remix.
